cloud/azure/main.tf

```diff
@@ -168,7 +167,7 @@ resource "azurerm_postgresql_server" "civiform" {
   auto_grow_enabled            = true
 
   # TODO: configure a subnet and restrict access only to the application servers.
-  public_network_access_enabled = true
+  public_network_access_enabled = false
```

Comment:
question (blocking): does this PR resolve this TODO?

Reply:
Pretty sure it does! It restricts access to only other servers inside the VNET.  Removed the comment.

Reply:
nice